andela/healthchecks-shockwave
# 150627366 send alert methods

# #150627366 Add Multiple Methods For Sending Alerts

# What does this PR do?

1. This PR allows users of the app to add integrations with SMS.
2. This PR allows users of the app to add integrations with Telegram.

# How should this be manually tested?

- cd into the healthchecks-shockwave folder
- Run the command 'python3 manage.py test'
- Ensure that your tests are running online as the server has to access other online servers.


# Any background context you want to provide?
This PR involves environment variables for twilio and telegram that needs to be placed before running or testing. The twilio functionality accesses an api that looks up numbers, while the telegram bot has to get data to validate a user's account. This necessitates the need for internet while validating of users on both services.

# What are the relevant pivotal tracker stories?
#150627366
